src/main/java/seedu/nuscents/parser/Parser.java

```diff
@@ -157,11 +162,61 @@ public static Expense parseExpense(String arguments) throws NuscentsException, P
             String date = extractValue("expense", arguments, DATE_PATTERN, false);
             String description = extractValue("expense", arguments, DESC_PATTERN, false);
             String additionalInformation = extractValue("expense", arguments, NOTE_PATTERN, true);
+            String category = extractValue("expense", arguments, CATEGORY_PATTERN, true);
```

Comment:
Since there's a new "category" field for transactions, perhaps there is a need to modify the help command message to reflect the expected use of this too.